README.md

```diff
@@ -1535,7 +1535,7 @@ Update Time, 5 active automations, webhooks.
 **[⬆️ Back to Top](#table-of-contents)**
 
 ## IDE and Code Editing
-
+   * [MarsX](https://www.marsx.dev/) - AI-powered dev tool  for building web and mobile apps out of micro-apps combining no-code and code interfaces.
```

Comment:
We only list entries once, but anyway this one has not launched so need to be removed.